lighthouse-core/test/audits/byte-efficiency/offscreen-images-test.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const UnusedImages =
+    require('../../../audits/byte-efficiency/offscreen-images.js');
+const TTIAudit = require('../../../audits/time-to-interactive.js');
+const assert = require('assert');
+
+/* eslint-env mocha */
+function generateRecord(resourceSizeInKb, startTime = 0, mimeType = 'image/png') {
+  return {
+    mimeType,
+    startTime,
+    resourceSize: resourceSizeInKb * 1024,
+  };
+}
+
+function generateSize(width, height, prefix = 'client') {
+  const size = {};
+  size[`${prefix}Width`] = width;
+  size[`${prefix}Height`] = height;
+  return size;
+}
+
+function generateImage(size, coords, networkRecord, src = 'https://google.com/logo.png') {
+  Object.assign(networkRecord || {}, {url: src});
+
+  const x = coords[0];
+  const y = coords[1];
+
+  const clientRect = {
+    top: y,
+    bottom: y + size.clientHeight,
+    left: x,
+    right: x + size.clientWidth,
+  };
+  const image = {src, networkRecord, clientRect};
+  Object.assign(image, size);
+  return image;
+}
+
+describe('OffscreenImages audit', () => {
+  const _ttiAuditOriginal = TTIAudit.audit;
+  before(() => {
+    const timeToInteractive = 2000000;
```

Comment:
`timeToInteractiveInμs`? :) took me a second and some looking at the TTI audit itself to realize this wasn't just chosen to be a really high TTI so it would be longer than any trace we normally take